### PR TITLE
add _redirects file

### PR DESCRIPTION
**Pull request by yulin-dong**

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -0,0 +1 @@
+/learn/platform-overview /learn/platform-overview/README
```
Comment (yulin-dong) [inline _redirects:1]: @dhrubabasu https://b97b68af.avalanche-docs.pages.dev/learn/platform-overview does not redirect. What did I do wrong?

Comment (dhrubabasu) [inline _redirects:1]:
```suggestion
/learn/platform-overview /learn/platform-overview/README 301
```

Comment (dhrubabasu) [reply _redirects:1]:
```suggestion
/learn/platform-overview /learn/platform-overview/README 301
```

Comment (dhrubabasu) [reply _redirects]: I think this is what you need to do ^

**Comment by yulin-dong** [comment]
@dhrubabasu https://2ee10fa4.avalanche-docs.pages.dev/learn/platform-overview/ still does not redirect with 301.

**Comment by yulin-dong** [comment]
Works now.

**Pull request merged**